dotnet/dynamodb/UpdateItemDataModel/UpdateItemDataModel.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+// SPDX-License-Identifier: MIT-0
+// snippet-start:[dynamodb.dotnet35.UpdateItemDataModel]
+using System;
+// using System.Collections.Generic;
+using System.Configuration;
+// using System.Globalization;
+// using System.Net.Sockets;
+using System.Threading.Tasks;
+
+using Amazon;
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.DataModel;
+
+namespace DynamoDBCRUD
+{
+    // DO NOT CHANGE THE TABLE ENTRY IN app.config!!!
+    [DynamoDBTable("CustomersOrdersProducts")]
+    public class Entry
+    {
+        [DynamoDBHashKey] // Partition key
+        public string ID
+        {
+            get; set;
+        }
+        [DynamoDBRangeKey] // Sort key
+        public string Area
+        {
+            get; set;
+        }
+        [DynamoDBProperty]
+        public int Order_ID
+        {
+            get; set;
+        }
+        [DynamoDBProperty]
+        public int Order_Customer
+        {
+            get; set;
+        }
+        [DynamoDBProperty]
+        public int Order_Product
+        {
+            get; set;
+        }
+        [DynamoDBProperty]
+        public long Order_Date
+        {
+            get; set;
+        }
+        [DynamoDBProperty]
+        public string Order_Status
+        {
+            get; set;
+        }
+    }
+
+    public class UpdateItemDataModel
+    {
+        public static async Task<Entry> UpdateTableItemAsync(IDynamoDBContext context, string id, string status)
+        {
+            // Retrieve the existing order
```

Comment:
Comments should be grammatically complete sentences--including punctuation (ending period in this case).

Reply:
Done.